Makefile

```diff
@@ -21,6 +21,9 @@ REPORTING_OPERATOR_IMAGE_REPO=$(DOCKER_BASE_URL)/origin-metering-reporting-opera
 REPORTING_OPERATOR_IMAGE_TAG=4.1
 METERING_OPERATOR_IMAGE_REPO=$(DOCKER_BASE_URL)/origin-metering-helm-operator
 METERING_OPERATOR_IMAGE_TAG=4.1
+METERING_ANSIBLE_OPERATOR_DOCKERFILE=Dockerfile.metering-ansible-operator
```

Comment:
Can you move this down to with the other dockerfile variables? under `METERING_OPERATOR_DOCKERFILE`

Reply:
Still not resolved